File: Patterns/2. Rectangle Patterns/18. Rectangle Alphabet Pattern/rap.py
l = int(input("Enter the Length : "))
b = int(input("Enter the Breadth : "))

for i in range(1,l+1):
    for j in range(1,b+1):
        print(chr(ord('A')+j-1),end = ' ')
    print()

'''
Output:

A B C D E F G 
A B C D E F G 
A B C D E F G 
A B C D E F G 
A B C D E F G 
'''

File: Patterns/2. Rectangle Patterns/18. Rectangle Alphabet Pattern/rap.cpp
#include <iostream>
using namespace std;
int main(){
    int l,b;
    cout<<"Enter the Length : ";
    cin>>l;
    cout<<"Enter the Brreadth : ";
    cin>>b;

    for(int i=1; i<=l; i++){
        for(int j=1; j<=b; j++){
            cout << char('A'+j-1) << " ";
        }
        cout <<'\n';
    }
}

/*
Output

A B C D E F G 
A B C D E F G 
A B C D E F G 
A B C D E F G 
A B C D E F G  

*/